sydent/db/version.py

```diff
@@ -0,0 +1,109 @@
+# -*- coding: utf-8 -*-
+
+# Copyright 2018 New Vector Ltd
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+import logging
+
+logger = logging.getLogger(__name__)
+
+
+class VersionStore:
+    def __init__(self, sydent):
+        self.sydent = sydent
+
+    def upgradeSchema(self):
+        curVer = self._getSchemaVersion()
+
+        if curVer < 1:
```

Comment:
should we do something to make new installations have schema v1 ?

Reply:
Yeah, this wasn't thought through very well. Hopefully this is more sensible now.